traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/Router.java

```diff
@@ -587,15 +589,15 @@ private URL getURL() {
 
                         sb.append(param.first);
                         sb.append("=");
-                        sb.append(param.second);
+                        sb.append(URLEncoder.encode(param.second, "UTF-8"));
```

Comment:
In Router.java we use 
`Uri.encode()` a couple of times to encode dates.

Let's stick to one solution: either `URLEncoder.encode()` or `Uri.encode()`

And if you add encoring here - it's better to remove all date encodings above in this file.

Reply:
I haven't notice dates encoding. I think it's better to do it on one place, so I've removed it from dates and added here with `Uri.encode()`